src/features/store.js

```diff
@@ -1,6 +1,6 @@
 import { configureStore } from '@reduxjs/toolkit';
 
-import { api } from './api';
+import { api } from './api/api.js';
```

Comment:
this would be recalled with:

```suggestion
import { api } from './api';
```